pastas/io/base.py

```diff
@@ -118,11 +118,17 @@ def _load_model(data: dict) -> Model:
         n = getattr(ps.noisemodels, data["noisemodel"].pop("class"))()
         ml.add_noisemodel(n)
 
-    # Add fit object to the model
+    # Add solver object to the model from pas-files < 1.3.0  TODO Deprecate
```

Comment:
Should we log a message recommending users to load and save their models to ensure pas files van be read with future versions of pastas?

Reply:
Yes , just committed the change.